### ISSUE TITLE
Nothing is updated to version 8.2.0 on Debian(Jessie)

### ISSUE THREAD

**Issue by phonec** [inverse-inc/packetfence#3798]
There is nothing files/packages to upgrade PacketFence from 8.1.0 to 8.2.0 during the upgrade procedure for a standalone server on Debian (Jessie) based system.
```
root@pf2:/# uname -a
Linux pf2 3.16.0-4-amd64 #1 SMP Debian 3.16.51-3 (2017-12-13) x86_64 GNU/Linux

root@pf2:/# cat /usr/local/pf/conf/pf-release
PacketFence 8.1.0

root@pf2:/# cat /usr/local/pf/conf/currently-at
PacketFence 8.0.1

root@pf2:/# apt update
Ign http://ftp.ua.debian.org jessie InRelease
Hit http://ftp.ua.debian.org jessie-updates InRelease
Hit http://ftp.ua.debian.org jessie Release.gpg
Hit http://ftp.ua.debian.org jessie-updates/main Sources
Get:1 http://ftp.ua.debian.org jessie-updates/main amd64 Packages/DiffIndex [11.8 kB]
Get:2 http://ftp.ua.debian.org jessie-updates/main Translation-en/DiffIndex [3,688 B]
Hit http://ftp.ua.debian.org jessie Release
Hit http://ftp.ua.debian.org jessie/main Sources
Get:3 http://security.debian.org jessie/updates InRelease [44.9 kB]
Hit http://ftp.ua.debian.org jessie/main amd64 Packages
Hit http://ftp.ua.debian.org jessie/main Translation-en
Hit http://inverse.ca jessie InRelease
Get:4 http://security.debian.org jessie/updates/main Sources [269 kB]
Hit http://inverse.ca jessie/jessie amd64 Packages
Get:5 http://security.debian.org jessie/updates/main amd64 Packages [619 kB]
Get:6 http://security.debian.org jessie/updates/main Translation-en [310 kB]
Ign http://inverse.ca jessie/jessie Translation-en_US
Ign http://inverse.ca jessie/jessie Translation-en
Fetched 1,258 kB in 1s (630 kB/s)
Reading package lists... Done
Building dependency tree
Reading state information... Done
All packages are up to date.


root@pf2:/# apt install packetfence
Reading package lists... Done
Building dependency tree
Reading state information... Done
packetfence is already the newest version.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
```

**Comment by nqb** [comment]
Hello @phonec,

Due to #3469, we postponed release of 8.2 for Debian. @fdurand is working on it, packages should be ready soon.

I let this issue opened to keep you inform

**Comment by danpoltawski** [comment]
I guess this issue can be closed now as 8.2.1 is in the repo.

**Comment by nqb** [comment]
Yes. Thanks.
